client/views/app/room.coffee

```diff
@@ -182,33 +182,25 @@ Template.room.helpers
 		status = Session.get 'user_' + username + '_status'
 		if status in ['online', 'away', 'busy']
 			return {username: username, status: status}
-		return
+		return {username: username, status: 'offline'}
 
 	roomUsers: ->
 		room = ChatRoom.findOne(this._id, { reactive: false })
 		users = []
-		onlineUsers = RoomManager.onlineUsers.get()
+		allUsers = RoomManager.allUsers.get()
 
 		for username in room?.usernames or []
-			if onlineUsers[username]?
-				utcOffset = onlineUsers[username]?.utcOffset
-				if utcOffset?
-					if utcOffset > 0
-						utcOffset = "+#{utcOffset}"
+			users.push
+				firstName: allUsers[username]?.firstName
+				lastName: allUsers[username]?.lastName
+				name: allUsers[username]?.name
+				username: username
+				status: allUsers[username]?.status
 
-					utcOffset = "(UTC #{utcOffset})"
-
-				users.push
-					username: username
-					status: onlineUsers[username]?.status
-					utcOffset: utcOffset
-
```

Comment:
Nice. The UTC time offset was kinda cool, possibly useful-ish, but it definitely cluttered things up over there.